src/mscorlib/shared/System/Numerics/Vector.cs

```diff
@@ -763,6 +767,36 @@ private Vector(ref Register existingRegister)
         {
             this.register = existingRegister;
         }
+
+        /// <summary>
+        /// Constructs a vector from the given span. The span must contain at least Vector'T.Count elements.
+        /// </summary>
+        [Intrinsic]
```

Comment:
I'm not sure the `[Intrinsic]` attribute is necessary/correct here.  I thought it only applied to members that the JIT needed to know about.  I don't think we need JIT interception here, do we?

Reply:
This method should have performance on par with the method that takes arrays. If the one that takes arrays needs to be treated as JIT intrinsic to get good performance, the one that takes Span needs to be treated as JIT intrinsic as well; and vice versa.

Reply:
Oops. I was too fast to remove the attribute and push a commit :-) Sorry. I think I will wait till the discussions are over.

Reply:
Apart from this discussion there was another similar review comment by @fiigii at [this place](https://github.com/dotnet/coreclr/pull/16733#discussion_r172058918). I'm not sure whether or not to retain the [Intrinsic] attribute. Appreciate inputs please.